src/components/indicators/indicators.js

```diff
@@ -97,28 +84,21 @@ define(['datetime', 'itemHelper', 'emby-progressbar', 'css!./indicators.css', 'm
         return '';
     }
 
-    function getCountIndicatorHtml(count) {
-        return '<div class="countIndicator indicator">' + count + '</div>';
-    }
-
-    function getChildCountIndicatorHtml(item, options) {
-        var minCount = 0;
-        if (options) {
-            minCount = options.minCount || minCount;
-        }
+    export function getChildCountIndicatorHtml(item, options) {
+        var minCount = options ? options.minCount : 0;
```

Comment:
What if we have `options` but no `options.minCount`?

Reply:
Seems like i forgot to add check for that mincount also